packages/amplify-provider-awscloudformation/src/aws-utils/aws-cfn.js

```diff
@@ -326,9 +326,24 @@ class CloudFormation {
     };
     const projectDetails = this.context.amplify.getProjectDetails();
     const { amplifyMeta } = projectDetails;
+
```

Comment:
use pagedAWSCall from https://github.com/aws-amplify/amplify-cli/blob/master/packages/amplify-provider-awscloudformation/src/aws-utils/paged-call.ts

Reply:
@attilah The pagedAWSCall uses "nextToken" as the param key whereas CFN expects it to be "NextToken" as @gakinson mentioned below, so might not be able to use the pagedAWSCall util function as is.

Reply:
@kaustavghosh06 getNextToken in that function is an accessor function for this reason, because it is varies across AWS APIs as they evolved. Like dynamodb is using something else as "NextToken", see usage here: https://github.com/aws-amplify/amplify-cli/blob/master/packages/amplify-provider-awscloudformation/src/aws-utils/DynamoDBService.ts#L41

Reply:
Got it. Made the change. Could you please review?